manager/__tests__/splits.test.js

```diff
@@ -0,0 +1,35 @@
+process.env.SPLITIO_EXT_API_KEY = 'test';
+process.env.SPLITIO_API_KEY = 'localhost';
+
+const request = require('supertest');
+const app = require('../../app');
+const { expectError } = require('../../utils/testWrapper/index');
+
+describe('splits', () => {
+  // Testing authorization
+  test('should be 401 if auth is not passed', async (done) => {
+    const response = await request(app)
+      .get('/splits');
+    expectError(response, 401, 'Unauthorized');
+    done();
+  });
+
+  test('should be 401 if auth does not match', async (done) => {
+    const response = await request(app)
+      .get('/splits')
+      .set('Authorization', 'invalid');
+    expectError(response, 401, 'Unauthorized');
+    done();
+  });
+
+  test('should be 200 and returns the splits added in YAML', async (done) => {
```

Comment:
Only as a nice to have and since it's just for testing... what if we add the yaml parser as a dev dependency and validate each object? 
Another option, just as good, is to validate some of the properties of each item (we can do a loop and receive expected values as a collection. Ideally besides knowing we have all the items being returned, we can validate the schema.

Reply:
I will consider this for the minor refactors!